view/template/trial.hbs

```diff
@@ -69,6 +69,23 @@
         </div>
         <div class="col-sm-12 col-md-7 col-lg-6 col-xl-6">
             <div class="card">
+                <div class = "modal-body">
+                    <form id = "check-compliance" method= "GET">
+                        <div class = "form-group">
+                            <div class = "col-md-4">
+                                <label>From date:-</label>
+                            </div>
+                            <input id="fromDate" name="fromDate" type="date" value="{{ fromDate }}" >
+                        </div>
+                        <div class = "form-group">
+                            <div class = "col-md-4">
+                                <label>To date:-</label>
+                            </div>
+                            <input id="toDate" name="toDate" type="date" value="{{ toDate }}">
+                        </div>
+                        <input type = "Submit" value = "Submit"></input>
```

Comment:
> Yes. After clicking on Sickel Cell trial the trial page renders fromDate as today and toDate as a week ahead with the do nut chart being displayed (for that week) without clicking on submit button. Next when you change dates and click submit do nut chart changes. :+1:


Right, first page load should have the chart :+1: 
I thought you were also planning on making each time the dates where change automatically refresh the chart without needing to click a button.